considered/rfc-saclar-initialization-checks.md

```diff
@@ -39,11 +39,11 @@ type whose completion is a scalar type) is initialized before it is read
 A read variable is considered to be local if either, the variable
 is library level and the read occurs directly in the elaboration of one of
 its enclosing packages or the variable is declared inside a subprogram, task, or
-entry and the expression is evaluated in the body or specification of the same
+entry and the expression is evaluated in the body or spec of the same
 unit.
 
 A variable is considered to be initialized if either it is a parameter of mode
-IN OUT, its type has a Default_Value aspect, an initial expression is supplied
+IN or IN OUT, its type has a Default_Value aspect, an initial expression is supplied
```

Comment:
well, I did not consider IN parameters to be variables, and we do not care about constants.